src/components/form/range/range_track.tsx

```diff
@@ -107,7 +107,7 @@ export const EuiRangeTrack: FunctionComponent<EuiRangeTrackProps> = ({
     styles.euiRangeTrack,
     disabled && styles.disabled,
     levels && !!levels.length && styles.hasLevels,
-    (tickSequence || ticks) && styles.hasTicks,
+    showTicks && styles.hasTicks,
```

Comment:
Do we still need to check that `tickSequence/ticks` are valid?
```suggestion
    showTicks && (tickSequence || ticks) && styles.hasTicks,
```